app/javascript/react/ansibleCatalog/catalogActions.test.js

```diff
@@ -0,0 +1,128 @@
+import fetchMock from 'fetch-mock';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './catalogActions';
+
+const mockRequest = (url, response) => {
+  fetchMock
+  .getOnce(url, response);
+};
+const resources = [{ id: 1, name: 'test' }];
+describe('Catalog Actions', () => {
+  let mockStore;
+  let middlewares;
+  let store;
+  beforeEach(() => {
+    middlewares = [thunk];
+    mockStore = configureMockStore(middlewares);
+    store = mockStore({});
+  });
+  afterEach(() => {
+    fetchMock.reset();
+    fetchMock.restore();
+  });
+
+  it('displays a catalog', () => {
+    const url = '/api/service_catalogs/?attributes=id,name&expand=resources&sort_by=name&sort_order=ascending';
+    mockRequest(url, {
+      resources,
+    });
+
+    return store.dispatch(actions.loadCatalogs()).then(() => {
+      const createdActions = store.getActions();
+      expect(createdActions).toMatchSnapshot();
```

Comment:
This is a interesting use of snapshots i love. Just make sure that the initial snapshot is correct. :+1: